schemas/schema.js

```diff
@@ -1458,7 +1458,7 @@ const schema = {
               ],
             },
             {
-              name: "dataPrivacySecurity[typesOfDataCollected]",
+              name: "doNoHarm[dataPrivacySecurity[typesOfDataCollected]]",
```

Comment:
@lacabra this line may address the bug that was breaking the build.
This is because `"dataPrivacySecurity": {"typesOfDataCollected": []}` has now been put under the `doNoHarm` key in the schema here

Reply:
Excellent, thank you